lib/Dialect/Arc/Transforms/ConvertToArcs.cpp

```diff
@@ -0,0 +1,364 @@
+//===- ConvertToArcs.cpp --------------------------------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
```

Comment:
Should this be in lib/Conversion instead of lib/Dialect/Arc/

Reply:
That's a good point. Yeah I guess this qualifies as HW-to-Arc conversion, even if the conversion is just partial.